.github/workflows/ nodejs.yaml

```
@@ -0,0 +1,40 @@
+name: Node.js CI
+
+on:
+  push:
+    paths:
+      - "src/**"
+      - ".github/workflows/nodejs.yaml"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x]
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
```

Do you want to use v2?

```suggestion
        uses: actions/setup-node@v2
```